server.go

```diff
@@ -18,13 +18,16 @@ func NewServer() *Server {
 }
 
 // Start executes a command in background.
-func (s *Server) Start(name string, arg ...string) error {
+func (s *Server) Start(name string, envs map[string]string, arg ...string) error {
```

Comment:
I think you don't need env here, it is not used.

Reply:
It's used in lines 28 to 30